docs/adr/0014-use-eclipse-orion-as-editor.md

```diff
@@ -0,0 +1,31 @@
+# Use Eclipse Orion as editor
+Editor should provide syntax highlighting and work in browser.
+
+## Considered Alternatives
+Compared the following cloud IDEs
+
+* [Eclipse Orion](https://wiki.eclipse.org/Orion)
+* [Eclipse Che](https://www.eclipse.org/che/)
+* [Codeanywhere](https://codeanywhere.com)
+* [Cloud9](https://c9.io/)
+* [Codiad](http://codiad.com/)
+* [Theia](https://github.com/theia-ide/theia)
+
+For more details see the [ultimate comparison](https://github.com/ultimate-comparisons/ultimate-webIDE-comparison).
```

Comment:
I would move this after line 23, because this provides more justification, doesn't it